NEWS.md

```diff
@@ -1,4 +1,4 @@
-# Version 2.0.0.9158
+# Version 2.0.0
```

Comment:
no worries about fixing this now, because the version bump action in GHA will change this again. we can just do it at the end before the CRAN release

Reply:
The version mismatch caused an error in the [version check job](https://github.com/openpharma/crmPack/actions/runs/10214239569/job/28261229637) of the CI.

Reply:
OK, then please adjust the `DESCRIPTION` version instead of the `NEWS` version